packages/backend/src/open_payments/wallet_address/service.ts

```diff
@@ -181,12 +188,13 @@ async function getWalletAddress(
 
 async function getOrPollByUrl(
   deps: ServiceDependencies,
-  url: string
+  url: string,
+  options: GetOrPollByUrlOptions = { mustBeActive: true }
```

Comment:
Since we primarily use this method when resolving the wallet address in Open Payments routes, we should expect it to always be active